.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -1,26 +1,30 @@
+-
+- ***
+
+## ✅ 체크리스트
 
 - [ ] 정상 동작 확인
-- [ ] 불필요한 코드 제거
+- [ ] 기존 기능 영향 여부 확인
 - [ ] 콘솔 에러 없음
+- [ ] 불필요한 코드 제거
 
 ---
```

markdownlint-cli2 (0.22.1)

[warning] Horizontal rule style
Expected: ***; Actual: ---

(MD035, hr-style)